# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/svg/crashtests/crashtests.list
# Commit: 78e7cb2837e7
# Full Hash: 78e7cb2837e7dec0ef55a2f5f383625335ad2520
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-08-17 15:01:15
# Description:
#   Bug 1848851 - Check target element is valid type before notifying r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D186358
# ==============================================================================

diff -r 970525aa04f1 -r 78e7cb2837e7 layout/svg/crashtests/crashtests.list
--- a/layout/svg/crashtests/crashtests.list	Thu Aug 17 10:06:10 2023 +0000
+++ b/layout/svg/crashtests/crashtests.list	Thu Aug 17 10:06:16 2023 +0000
@@ -259,3 +259,4 @@
 load 1826444-1.html
 load 1831419.html
 load 1840195-1.html
+load 1848851.html
